home-server-stub/pom.xml

```diff
@@ -11,7 +11,111 @@
 
     <packaging>jar</packaging>
 
+    <properties>
+        <yaml.name>openapi.yaml</yaml.name>
+    </properties>
+
     <artifactId>home-server-stub</artifactId>
```

Comment:
Let's rename the module to `home-server-generator`

Reply:
Ok, noted

Reply:
Wait, what should we rename here? openapi.yaml to home-server-generator.yaml?